### PR TITLE
ZENKO-4363 add azure archive tests

### PR DESCRIPTION
**Pull request by welansari**
Issue: ZENKO-4363

### CODE REVIEW

File: tests/ctst/features/azureArchive.feature [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        # |               Suspended |
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Respect maximum number of objects per archived Tar
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And an object "obj1" of size <objectSize> bytes
```
Comment (francoisferrand) [inline tests/ctst/features/azureArchive.feature]: could we add another parameter to this rule:

> And <count> objects "obj" of size <objectSize> bytes

where we could either run with multiple counts (→ see different behaviors), or at least avoid repeating 5 times the same step/condition ?

File: tests/ctst/features/azureArchive.feature [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    And manifest and tar containing object "obj5" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+    And manifest and tar containing object "obj6" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+
+    Examples:
+        | versioningConfiguration | objectSize | packObjectCount |
+        |           Non versioned |          1 |               3 |
+        |               Versioned |          1 |               3 |
+        # |               Suspended |          1 |               3 |
```
Comment (francoisferrand) [inline tests/ctst/features/azureArchive.feature]: should we have another config for "_non-versioned object in a versioned bucket_" ?

Comment (welansari) [reply tests/ctst/features/azureArchive.feature]: i think we should wait for the follow up test ticket before adding this, as that case will change the semantics of the tests, we might want to make it it's own scenario

File: .github/scripts/end2end/configs/zenko.yaml [hunk 3]
```diff
@@ -94,6 +94,15 @@ spec:
       e2e-cold:
         debug: "true"
         command-timeout: "60s"
+    serverConfiguration:
+      ${AZURE_ARCHIVE_BACKEND_DESTINATION_LOCATION}:
```
Comment (francoisferrand) [inline .github/scripts/end2end/configs/zenko.yaml]: with latest ZKOP :

```
server:
   tar:
      ...
   azure:
      ...
```

File: tests/ctst/features/azureArchive.feature (hunk 2 above)
Comment (francoisferrand) [reply tests/ctst/features/azureArchive.feature]: ok, please add this to the followup ticket so we don't forget

Comment (welansari) [reply tests/ctst/features/azureArchive.feature]: done

File: solution/deps.yaml [hunk 4]
```diff
@@ -94,7 +94,7 @@ vault:
 zenko-operator:
   sourceRegistry: registry.scality.com/zenko-operator
   image: zenko-operator
-  tag: 1.5.21
+  tag: 1.5.24
```
Comment (francoisferrand) [inline solution/deps.yaml]: not needed, just rebase to get latest zkop...

**Review by francoisferrand — APPROVED** [review]

**Review by KillianG — APPROVED** [review]

**Pull request merged**